frb_rust/src/handler.rs

```diff
@@ -298,6 +306,21 @@ impl Error {
     }
 }
 
+impl IntoDart for Error {
+    fn into_dart(self) -> allo_isolate::ffi::DartCObject {
+        match self {
+            Error::CustomError(e) => e(),
```

Comment:
To maintain compatibility, maybe we should add another extra test in pure_dart/dart/main.dart:

When a function throws a `anyhow` error (you know, the old ways before this pr), we should assert that, we can see error message AND error stack traces.

Reply:
For anyhow to be an FrbException, we need to implement this concept here:

impl IntoDart for anyhow::Error {
    fn into_dart(self) -> DartCObject {
        self.to_string().into_dart()

but of course this is  in allo-isolate. So what is your suggestion?

Reply:
Contributing to allo-isolate sounds ok to me.

Btw here are two things: 

* (maybe) contribute to allo-isolate
* add a test (everyone wants to see that error stacktrace, because before this pr they saw it, so we should not regress)

Reply:
The anyhow seems really hard unless I change `anyhow::Result` by a type `AnyhowResult`, what do you think? Somehow it needs to have a name that I can parse, but `Result` is not good

Reply:
> Somehow it needs to have a name that I can parse, but Result is not good

is it possible to parse `anyhow::Result` - we may collect `::` along the way

> change anyhow::Result by a type AnyhowResult

That may make users a bit confused imho

Reply:
We don't receive the entire path when parsing:

`Result<u32, CustomStructError>`

gives

`Path(SupportedPathType { ident: Ident(Result), generic: [Path(SupportedPathType { ident: Ident(u32), generic: [], is_exception: false }), Path(SupportedPathType { ident: Ident(CustomStructError), generic: [], is_exception: false })], is_exception: false })`

Reply:
But when parsing `anyhow::Result<.., ..>` we do get "anyhow"?

Reply:
yes,

```rust
pub fn return_err_2() -> anyhow::Result<VeryCustomStruct> {
    Err(anyhow!(
        "return_err() is called, thus deliberately return Err"
    ))
}
```

gives

`Path(SupportedPathType { ident: Ident(Result), generic: [Path(SupportedPathType { ident: Ident(VeryCustomStruct), generic: [], is_exception: false })], is_exception: false })`

Reply:
I mean, no, we don't

Reply:
Hmm... So we may need to extend the parser a bit?

Or, a very very hacky way: if Result only has one generic arg it is treated as anyhow (really hacky!)